LoggerInterface/src/FileLogger.java

```diff
@@ -0,0 +1,24 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class FileLogger implements Logger {
+    @Override
+    public void log(String messeage) {
+        System.out.println("Please give a file name");
+        String fileName=getText();
+        System.out.println("Please give a message");
+        messeage=getText();
+        try (FileWriter writer = new FileWriter("C:/"+fileName+".txt")) {
+            writer.write(messeage);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private String getText(){
+        Scanner scanner=new Scanner(System.in);
+        String text = scanner.nextLine();
+        return  text;
+    }
```

Comment:
The idea itself is creative - but how am I going to use this as an external API user? I want (and need) to decide the way of logging compile time, not runtime! Just have a field called `filePath` and have an all-args constructor and initialize the variable there.

Reply:
I see nothing changed here - reopening

Reply:
So maybe I misunderstood your comment. 
If I am right, the issue was the file path was hardcoded, so  now it can be given by the user.
Or that is the problem? I should make a String variable FilePath where I can give the path before runtime?